test/scrabble_test.rb

```diff
@@ -5,9 +5,13 @@
 require 'pry'
 
 class ScrabbleTest < Minitest::Test
+
+  def setup
+    @scrabble = Scrabble.new
+  end
+
   def test_it_can_score_a_single_letter
-    skip
-    assert_equal 1, Scrabble.new.score("a")
-    assert_equal 4, Scrabble.new.score("f")
+    assert_equal 1, @scrabble.score("a")
+    assert_equal 4, @scrabble.score("f")
```

Comment:
Use your tests! Per the comment above, would've liked to see this test passing and tests for the earlier pieces rather than starting to build the later functionality.